README.md

```diff
@@ -76,7 +76,8 @@ Name                                       | Description
 -------------------------------------------|--------------------------------------------------------------------------------------------------
 config.my-cnf                              | Path to .my.cnf file to read MySQL credentials from. (default: `~/.my.cnf`)
 log.level                                  | Logging verbosity (default: info)
-log_slow_filter                            | Add a log_slow_filter to avoid exessive MySQL slow logging.  NOTE: Not supported by Oracle MySQL.
+exporter.lock_wait_timeout                 | Set a lock_wait_timeout on the connection to avoid long metadata locking. (default: 2 seconds)
+exporter.log_slow_filter                   | Add a log_slow_filter to avoid exessive MySQL slow logging.  NOTE: Not supported by Oracle MySQL.
```

Comment:
Just a bonus question, what's the purpose of this flag in the exporter? To overwrite global `log_slow_filter` or to avoid query logging of exporter own queries?

This was a bit unclear to me from above description but after I found https://github.com/prometheus/mysqld_exporter/pull/60 the sentence "to avoid slow query logging of scrapes" makes it more clear.

Reply:
It's a session flag to avoid the exporter logging its own queries.  Yes, I think your description of the flag is better.